### DISCUSSION TITLE
Problems getting Kong Manager seeing admin api

### DISCUSSION THREAD

**Discussion by scm7mae** [Kong/kong#12751] [Help] [answered]
Hi All,

Really hope you can help.  I've been battling with this for a month or so, and can't seem to get it to work, so hoping someone here can identify my errors, and point me in the right direction.

I've managed to install Kong ok within k3s using helm.  The issue I have is every time I try to access the Manager GUI, I get an error saying it cannot connect to the admin api.

I've tried a couple of different ways, but still cannot get it to work, either, seeing a cross site scripting error, or cannot resolve address problem.  I've installed kong using the following command: 

helm install kong-with-postgres --namespace default kong/kong    \
--set ingressController.installCRDs=false    \
--set env.database=postgres    \
--set admin.enabled=true    \
--set admin.http.enabled=true    \
--set env.pg_host=postgres-kong-service.default.svc.cluster.local    \
--set env.admin_api_url=http://kong-with-postgres-admin.default.svc.cluster.local:8001    \
--set env.pg_user=postgres    \
--set env.pg_password=kong   \
 --set env.pg_port=5432    \
--set env.pg_database=postgres

However, when I then access the admin interface, I see the following error in the dev tools of the browser:

![image](https://github.com/Kong/kong/assets/8223835/1958a681-7180-4eaf-956a-ef527fdeee1b)

What am I doing in terms of config and installation that is incorrect?

Appreciate the help in advance.

**Comment by randmonkey** [comment, marked as answer]
From the debug message in your browser, the `kong-with-postgres-admin.deafult.svc.cluster.local` could not be resolved. This is a domain that can only be accessed inside your k8s cluster, so the environment where you open your browser cannot access it.

**Reply by scm7mae** [reply]
Thanks. I’ve resolved the issue.

**Comment by Gustavo-Coutinho** [comment]
@randmonkey but from where exactly should we be accessing kong manager interface? 

From my understanding, the Kong Manager itself should access the Admin API, and not the user's browser that should access the Admin API.
